### PR TITLE
Implemented basic server logic

### PR DESCRIPTION
**Pull request by stasya72008**

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    return json.dumps(users)
+
+
+@app.route('/users', methods=['POST'])
+def add_user():
+    users.append(USER_TO_ADD)
+    return json.dumps(users)
+
+
+@app.route('/users/<user_id>', methods=['DELETE'])
```
Comment (denyskulykov) [inline app/routes.py]: Please, use <int: user_id>

File: app/routes.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+def delete_user(user_id):
+    for user in users:
+        if user['id'] == int(user_id):
+            users.remove(user)
+        else:
+            print ('My friends are gonna be there too')
```
Comment (denyskulykov) [inline app/routes.py]: Strange...
You have a lot of print with 'my friends...'

File: app/routes.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+def get_user_tasks(user_id):
+    user_tasks = list()
+    for task in tasks:
+        if task['user_id'] == int(user_id):
+            user_tasks.append(task)
+
+    return json.dumps(user_tasks)
+
+
+@app.route('/tasks/<task_id>', methods=['GET'])
```
Comment (denyskulykov) [inline app/routes.py]: the same about <int: ...> and please change everywhere

File: app/data/tasks.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": 1,
+    "user_id": 1,
```
Comment (denyskulykov) [inline app/data/tasks.json]: Maybe 'autor'

File: app/data/tasks.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": 1,
+    "user_id": 1,
+    "name": "Livin' easy",
+    "date": "today",
+    "status": "active"
```
Comment (denyskulykov) [inline app/data/tasks.json]: Please, add fields for task like
'from', 'to' - date
'duration'  - time (hours)
also 
'number' - id in list, witch be show for user

File: app/__init__.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+from flask import Flask
```
Comment (denyskulykov) [inline app/__init__.py]: I think, it' not a good plase for this action
Olease move it to logic modul, app/routes.py

File: main.py [hunk 7]
```diff
@@ -0,0 +1 @@
+from app import app
```
Comment (denyskulykov) [inline main.py]: Do we need this file?

File: app/data/tasks.json (hunk 4 above)
Comment (stasya72008) [reply app/data/tasks.json]: Or maybe 'aut**h**or'?..

File: app/routes.py (hunk 2 above)
Comment (stasya72008) [reply app/routes.py]: Better say "you'll", cause I won't have it, until code is runned

File: app/data/tasks.json (hunk 5 above)
Comment (stasya72008) [reply app/data/tasks.json]: 'witch be show' - please, enter it into Google Translate :)

File: app/__init__.py (hunk 6 above)
Comment (stasya72008) [reply app/__init__.py]: it'**s**
pla**c**e

File: app/routes.py (hunk 2 above)
Comment (denyskulykov) [reply app/routes.py]: :P

File: app/__init__.py (hunk 6 above)
Comment (denyskulykov) [reply app/__init__.py]: ? 
you mean - "place it to ..."

File: app/__init__.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
+from flask import Flask
+from app import routes
+
```
Comment (denyskulykov) [inline app/__init__.py]: Please, rename folder like: **rest_server** or **rest_api**

File: app/__init__.py (hunk 6 above)
Comment (stasya72008) [reply app/__init__.py]: No, I mean you have a typo in "good place for..."

**Pull request merged**